packages/app-admin-ui/client/pages/List/Management.js

```diff
@@ -17,74 +17,75 @@ export const ManageToolbar = styled.div(({ isVisible }) => ({
   marginTop: gridSize,
   visibility: isVisible ? 'visible' : 'hidden',
 }));
-const SelectedCount = styled.div({
-  color: colors.N40,
-  marginRight: gridSize,
-});
 
-export default function ListManage(props) {
-  const { onDeleteMany, onUpdateMany, selectedItems } = props;
-  const [deleteModalIsVisible, setDeleteModal] = useState(false);
-  const [updateModalIsVisible, setUpdateModal] = useState(false);
+const SelectedCount = styled.div`
```

Comment:
Why did we change to template functions here? We use object notation throughout most of the codebase. I'd rather be consistent.

Reply:
There's no consistent usage of `styled` object notation vs template literals. Lot of both. I've been gradually switching to the latter as I've come across it. Haven't touched the function-based ones since I only yesterday realized that they can return a template literal too 😬